### PR TITLE
Be da 0409

### PR DESCRIPTION
**Pull request by habibdevsy**

### CODE REVIEW

File: backend/src/Manager/Subscription/SubscriptionManager.php [hunk 1]
```diff
@@ -318,4 +318,9 @@ public function updateSubscription(SubscriptionUpdateByAdminRequest $request)
 
        return "ok";
     }
+
+    public function getUserId(int $storeOwnerProfileId): ?StoreOwnerProfileEntity
```
Comment (ralsalloum) [inline backend/src/Manager/Subscription/SubscriptionManager.php]: function name could be updated to more useful one.

File: backend/src/Response/Admin/StoreOwnerSubscription/AdminStoreSubscriptionResponse.php [hunk 2]
```diff
@@ -46,4 +46,18 @@ class AdminStoreSubscriptionResponse
...
      */
     public $captainOffers;
+
+    public int $packageCarCount;
+
+    public int $packageOrderCount;
+
+    public int $packageExpired;
+
+    public string $packageNote;
```
Comment (ralsalloum) [inline backend/src/Response/Admin/StoreOwnerSubscription/AdminStoreSubscriptionResponse.php]: package note may be null, so we have to set null as probably return type for packageNote beside string.

File: backend/src/Service/Subscription/SubscriptionService.php [hunk 3]
```diff
@@ -624,5 +624,24 @@ public function updateSubscription(SubscriptionUpdateByAdminRequest $request)
     {
         return $this->subscriptionManager->updateSubscription($request);
     }
+
+    public function packageBalanceForAdminByStoreOwnerProfileId(int $storeOwnerProfileId)
```
Comment (ralsalloum) [inline backend/src/Service/Subscription/SubscriptionService.php]: function return type is missing.

**Review by ralsalloum — APPROVED** [review]
Reviewed. @habibdevsy please check my notes.

**Comment by habibdevsy** [comment]
Done @ralsalloum

**Review by ralsalloum — APPROVED** [review]

**Pull request closed without merging**